data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -263,6 +264,7 @@
     (
         GeneralizeStateByPopulation,),
     (GeneralizeCopeInsuranceAnswers,),
+    (GeneralizeSexGenderConcepts,),
```

Comment:
I'm not sure we need to include this in the Clean cdr yet.  This is currently done by the the legacy Registered Tier code and we don't want to try to do this twice.  i would suggest commenting this out for now.